aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -78,11 +77,13 @@ void check_allany_for_meta(
 }
 
 TORCH_META_FUNC2(all, dim)(const Tensor& self, int64_t dim, bool keepdim) {
-  check_allany_for_meta(*this, "all", self, dim, keepdim);
+  this->dim = maybe_wrap_dim(dim, self.dim());
```

Comment:
mmmm. Let's enforce the convention that all members have trailing underscores, it will make it more obvious what's going on here